### PR TITLE
Switch format for RPC proofs_of_inclusion for DL offers

### PR DESCRIPTION
**Pull request by Quexington**

### CODE REVIEW

**Review by altendky — DISMISSED** [review]

File: chia/data_layer/data_layer_wallet.py [hunk 1]
```diff
@@ -1272,10 +1272,9 @@ async def finish_graftroot_solutions(offer: Offer, solver: Solver) -> Offer:
                         asserted_root: Optional[str] = None
                         proofs_of_inclusion = []
                         for value in values:
-                            for root in solver["proofs_of_inclusion"].info:
-                                proof: Tuple[int, List[bytes32]] = tuple(  # type: ignore
-                                    solver["proofs_of_inclusion"][root]
-                                )
+                            for proof_of_inclusion in solver["proofs_of_inclusion"]:
+                                root = proof_of_inclusion[0]
+                                proof = tuple(proof_of_inclusion[1:])
```
Comment (altendky) [inline chia/data_layer/data_layer_wallet.py:1277]: This is still just playing on the `Any` nature of the underlying data, but might satisfy things.

```suggestion
                                proof: Tuple[int, List[bytes32]] = tuple(proof_of_inclusion[1:])
```

Comment (altendky) [reply chia/data_layer/data_layer_wallet.py]: This is wrong, I should have removed it before heading out...  sorry.

File: chia/data_layer/data_layer_wallet.py [hunk 2]
```diff
@@ -1272,10 +1272,9 @@ async def finish_graftroot_solutions(offer: Offer, solver: Solver) -> Offer:
                         asserted_root: Optional[str] = None
                         proofs_of_inclusion = []
                         for value in values:
-                            for root in solver["proofs_of_inclusion"].info:
-                                proof: Tuple[int, List[bytes32]] = tuple(  # type: ignore
-                                    solver["proofs_of_inclusion"][root]
-                                )
+                            for proof_of_inclusion in solver["proofs_of_inclusion"]:
+                                root = proof_of_inclusion[0]
+                                proof: Tuple[int, List[bytes32]] = tuple(proof_of_inclusion[1:])
```
Comment (altendky) [inline chia/data_layer/data_layer_wallet.py:1277]: Is the `str` correct here?  Or would it have been processed into a `bytes32`?  Or would that depend on it having a leading `0x` or not?

```suggestion
                                root: str = proof_of_inclusion[0]
                                proof: Tuple[int, List[bytes32]] = (proof_of_inclusion[1], proof_of_inclusion[2])
```

Comment (Quexington) [reply chia/data_layer/data_layer_wallet.py:1277]: Yeah str is correct right now

**Comment by altendky** [comment]
Rerunning with the other fix for the broken tests.

**Review by altendky — APPROVED** [review]

**Pull request merged**